pkg/cmd/server/start/start_node.go

```diff
@@ -280,8 +274,13 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 
 	config.EnsureVolumeDir()
 	config.EnsureDocker(docker.NewHelper())
+	plugin, endpointFilter := InitSDNPlugin(config, nodeConfig)
```

Comment:
> Thoughts on where to put FilteringEndpointsConfigHandler then? Maybe api/types.go?

as in pkg/sdn/api/types.go? Makes sense to me. (I just put it wherever was simplest originally.)